src/main/webAuthnClient.ts

```diff
@@ -92,42 +98,75 @@ export default class WebAuthnClient {
     return this.http.get<DeviceCredential[]>(this.registrationUrl, { accessToken })
   }
 
+  private isDiscoverable(params: LoginWithWebAuthnParams): params is DiscoverableLoginWithWebAuthnParams {
+    return (params as DiscoverableLoginWithWebAuthnParams).conditionalMediation !== undefined
```

Comment:
Alors je sais que c'est courant de voir cette syntaxe dans notre code, mais c'est une mauvaise façon de faire car une variable "undefined" pourait très bien avoir été définie (ex: `var undefined = "oops"` et rendre ce test caduque.
La bonne manière de faire est d'utiliser le mot clé `typeof` :

```suggestion
    return typeof (params as DiscoverableLoginWithWebAuthnParams).conditionalMediation !== "undefined"
```